cypress/objects/taskPage.js

```diff
@@ -0,0 +1,36 @@
+class Tasks {
+    getTasks() {
+        return cy.get('[href="/tasks"]');
+    }
+    getButton() {
+        return cy.get('.btn');
```

Comment:
need more accurate name than generic getButton, I need to understand what I get here